tests/forms_tests/tests/test_media.py

```diff
@@ -106,7 +108,7 @@ class Media:
         class MyWidget3(TextInput):
             class Media:
                 css = {
-                    'all': ('/path/to/css3', 'path/to/css1')
+                    'all': ('path/to/css1', '/path/to/css3')
```

Comment:
Sadly yes. If you expand the whole test, you will find, that most tests here have the same assets but in the opposite order. Since I added a warning, this warning will break the tests.
I capture all warnings in the tests, but since those are unit tests, I figured it they should be tailored to each case. Testing the warning case multiple times, seemed to miss that point.